test/helpers/kubectl.go

```diff
@@ -153,6 +156,40 @@ var (
 		"devices":                     "", // Override "eth0 eth0\neth0"
 	}
 
+	aksHelmOverrides = map[string]string{
+		"image.repository":          "mcr.microsoft.com/oss/cilium/cilium",
+		"image.tag":                 "1.12.1",
```

Comment:
Could you parameterize these `repository` + `tag` references? I think it makes sense to be able to launch the testsuite against target development images like this, but given that the Cilium CNCF project doesn't have control over these Microsoft repositories, it is a bit odd to hardcode these references into the code tree. Also it would be useful to not have to update the version string here every time that we want to run tests against a different version.

In other cases, we have just defined these via environment variables and passed them in as overrides.

Reply:
Sure, make sense

Reply:
Yeah that's what I had in mind. I suspect there's already some way to do this already, because the existing cloud jobs would need to pull the image from somewhere (which is typically from cilium/cilium-ci).